src/sentry/utils/performance_issues/detectors/large_payload_detector.py

```diff
@@ -55,6 +60,10 @@ def _store_performance_problem(self, span) -> None:
                 "cause_span_ids": [],
                 "offender_span_ids": offender_span_ids,
                 "op": "http",
+                "transaction_name": self._event.get("description", ""),
+                "repeating_spans": get_span_evidence_value(span),
+                "repeating_spans_compact": get_span_evidence_value(span, include_op=False),
+                "num_repeating_spans": str(len(offender_span_ids)),
```

Comment:
There's a bug here where `offender_span_ids` is just the offending span string and not an array, the fix is here: https://github.com/getsentry/sentry/pull/48550/